test/features/pagecontent/save_api.js

```diff
@@ -418,6 +418,7 @@ describe('page save api', function() {
                     uri: htmlUri,
                     headers: {
                         'x-forwarded-for': '123.123.123.123',
+                        'x-client-ip': '123.123.123.123',
```

Comment:
Why are both set here? Shouldn't we set only `x-client-ip` and then let RB copy it as `XFF` ?